src/openforms/emails/tasks.py

```diff
@@ -23,13 +24,15 @@ def __init__(self, since: datetime) -> None:
     def get_context_data(self) -> dict[str, Any]:
         failed_emails = collect_failed_emails(self.since)
         failed_registrations = collect_failed_registrations(self.since)
+        failed_prefill_plugins = collect_failed_prefill_plugins(self.since)
 
-        if not (failed_emails or failed_registrations):
+        if not (failed_emails or failed_registrations or failed_prefill_plugins):
```

Comment:
I'm wondering if this code path is necessary: if we remove it, the next return statement will be something like:

```python
{
    "failed_emails": {},
    "failed_registrations": {},
    "failed_prefill_plugins": {},
}
```

And the rendered template content will be the same as the template checks for truthy values on these variables